Keywords/pages/Page_Licensure_Unit_Appeals.groovy

```diff
@@ -20,38 +22,120 @@ import com.kms.katalon.core.webui.keyword.WebUiBuiltInKeywords as WebUI
+	private TestObject inputPhoneExt			= findTestObject('Page_Licensure_Unit_Appeals/input_Applicant_Phone_Ext')
+	private TestObject inputAddressL1			= findTestObject('Page_Licensure_Unit_Appeals/input_Applicant_Address_L1')
+	private TestObject inputAddressL2			= findTestObject('Page_Licensure_Unit_Appeals/input_Applicant_Address_L2')
+	private TestObject inputAddressL3			= findTestObject('Page_Licensure_Unit_Appeals/input_Applicant_Address_L3')
+	private TestObject inputZip					= findTestObject('Page_Licensure_Unit_Appeals/input_Applicant_Zip')
+	private TestObject textareaAppealReason		= findTestObject('Page_Licensure_Unit_Appeals/textarea_ReasonForAppeal')
+	private TestObject h1AppealSubmitted		= findTestObject('Page_Licensure_Unit_Appeals/h1_Your appeal has been submitted')
 
-	private TestObject inputAppealCode						= findTestObject('Page_Licensure Unit Appeals/input_Appeal Code')
-	private TestObject buttonNext							= findTestObject('Page_Licensure Unit Appeals/button_Next')
-	private TestObject inputEmail							= findTestObject('Page_Licensure Unit Appeals/input_Applicant Email')
-
+	/*************
+	 * Inputs
+	 *************/	
 	@Keyword
 	def enterAppealCode (String appealCode) {
 		WebUI.waitForElementVisible(inputAppealCode, 0)
 		WebUI.sendKeys(inputAppealCode, appealCode)
 	}
 
 	@Keyword
-	def clickOnNextButton () {
-		WebUI.waitForElementVisible(buttonNext, 0)
-		WebUI.click(buttonNext)
+	def enterUpdatedPhone (String phone) {
+		WebUI.waitForElementVisible(inputPhone, 0)
+		WebUI.setText(inputPhone, phone)
+	}
+	@Keyword
+	def enterUpdatedPhonExt (String ext) {
+		WebUI.waitForElementVisible(inputPhoneExt, 0)
+		WebUI.setText(inputPhoneExt, ext)
+	}
+
+	@Keyword
+	def enterUpdatedAddressL1 (String address) {
+		WebUI.waitForElementVisible(inputAddressL1, 0)
+		WebUI.setText(inputAddressL1, address)
+	}
+	@Keyword
+	def enterUpdatedAddressL2 (String address) {
+		WebUI.waitForElementVisible(inputAddressL2, 0)
+		WebUI.setText(inputAddressL2, address)
+	}
+
+	@Keyword
+	def enterUpdatedAddressL3 (String address) {
+		WebUI.waitForElementVisible(inputAddressL3, 0)
+		WebUI.setText(inputAddressL3, address)
+	}
+
+	@Keyword
+	def enterUpdatedZip (String zip) {
+		WebUI.waitForElementVisible(inputZip, 0)
+		WebUI.setText(inputZip, zip)
+	}
+
+	@Keyword
+	def enterAppealReason (String text) {
+		WebUI.waitForElementVisible(textareaAppealReason, 0)
+		WebUI.sendKeys(textareaAppealReason, text)
+	}
+
+	/*************
+	 * Buttons
+	 *************/
+	@Keyword
+	def clickOnNextBtn () {
+		WebUI.waitForElementVisible(btnNext, 10)
+		WebUI.click(btnNext)
+	}
+
+	@Keyword
+	def clicOnCloseModalBtn () {
+		WebUI.waitForElementVisible(btnModalClose, 10)
+		WebUI.click(btnModalClose)
+	}
+
+	@Keyword
+	def clicOnModalSubmitBtn () {
+		WebUI.waitForElementVisible(btnModalSubmit, 10)
+		WebUI.click(btnModalSubmit)
+	}
+	
+	@Keyword
+	def clicOnSubmitBtn () {
```

Comment:
Please, update the keyword's name. It should be "click" not "clic"

Reply:
Done